src/scripts/app.ts

```diff
@@ -132,57 +137,63 @@ function createToFindItem() {
 
 const checkElements = (event: Event) => {
 	const clickedImg = event.target as HTMLElement;
-	const imgFind = document.querySelector('.image-to-find') as HTMLImageElement;
 
+	const imgFind = document.querySelector('.image-to-find') as HTMLImageElement;
 	const clickedSrcValue = clickedImg.getAttribute('src');
 	const srcToFindValue = imgFind.getAttribute('src');
 	const clickedIdValue = clickedImg.getAttribute('id');
 	const idToFind = imgFind.getAttribute('id');
 
-	if (attemptCounter >= maxAttempts) {
-		showToast('error', 'Brak dalszych prób');
+	if (ATTEMPT_COUNTER >= MAX_ATTEMPTS) {
+		showToast('error', 'No further attempts, Try Again');
 		return;
 	}
```

Comment:
No i tutaj pewnie wdrożysz jeszcze jakiś modal "przegrałeś, Twój wynik to XY" + przycisk "Zagraj ponownie" który przeładuje gre na nowo.